RPS src/app.js

```diff
@@ -0,0 +1,53 @@
+import compPlay from './RPS.js';
+import rpsCompair from './RPScomp.js';
+
+const fight = document.getElementById('fight');
+
+const playerImg = document.getElementById('playerMove');
```

Comment:
your code should all be in the src folder, and if you want further organization, create new folders within the src folder.  Also, naming wise, you should never have spaces within file or folder names, use rps_src or rpsSrc